### DISCUSSION TITLE
How to override a model ?

### DISCUSSION THREAD

**Discussion by maxime-aubry** [prisma/prisma#25467] [Q&A]
### Question

> i am creating a common library for every nestjs projects in will make.
I would like this common project handles users and authentification.
So, i have a schema.prisma with User table inside.

How Can i do to override this user table in schéma.prisma file in another schema to add relations, other properties ?...

I saw it IS possible multiple schema.prisma files, but i did not found the way to override tables.

Thanks a lot.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @maxime-aubry 

While it is possible to have multiple `.prisma` schema files, there isn't a native way to directly **override** a table (model) in one schema using another. Prisma's design is based on a single schema file as the source of truth, and the models are compiled together into one set of Prisma Client types. Prisma doesn’t natively support schema inheritance or table overriding across multiple schema files. We do have a [feature request](https://github.com/prisma/prisma/issues/12604) for supporting schema inheritance. You can add your use-case and leave a 👍 to help our engineers prioritize it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
